### DISCUSSION TITLE
Blazorise & Maui Blazor

### DISCUSSION THREAD

**Discussion by walidmasa** [Megabit/Blazorise#5229] [General]
the blazoris components are they functional with the maui blazor project

**Comment by David-Moreira** [comment]
Hello,

Blazorise components are developed for Blazor. So they should work just fine in MAUI Blazor since it uses the same engine or at least I think it does, anything that renders in regular Blazor, is usually expected to render the same in MAUI Blazor. I do not think there are known limitations.

Now, we did have some reports in the past to do with trimming/aot removing stuff it probably should not, we've reported it to the .NET team, but I think these might not be fixed yet, so beware of that.
